addons/account/static/src/components/account_payment_field/account_payment.xml

```diff
@@ -69,33 +69,36 @@
                 <div>
                     <table>
                         <tr>
-                            <td><strong>Amount: </strong></td>
-                            <td>
+                            <td class="fw-bolder">Amount: </td>
+                            <td class="o_popover_field_content">
```

Comment:
could you try with a ps-1 instead of making a new css class for this ? Let's try to maximize the use of bootstrap